patient-appointment-management/.env

```diff
@@ -0,0 +1,88 @@
+# description: Customer name to be displayed in SMS.
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_NAME=
+
+# description: Customer short name. Must be all lowercase one word without any spaces.
+# format: text
+# required: true
+# configurable: true
+APPLICATION_CUSTOMER_CODE=
+
+# description: Customer EHR endpoint url for communication from Twilio via HTTP POST
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_EHR_ENDPOINT_URL=
+
+# description: Twilio phone number to send SMS from. Your patient will see this number.
+# format: phone_number
+# required: true
+# configurable: true
+TWILIO_PHONE_NUMBER=
+
+# description: Reminder outreach window start time local timezone (HHMM) inclusive. default 0000
+# format: text
+# required: true
+# configurable: true
+# default: 0000
+REMINDER_OUTREACH_START=
+
+# description: Reminder outreach window end time local timezone (HHMM) exclusive, default 2400
+# format: text
+# required: true
+# configurable: true
+# default: 2400
+REMINDER_OUTREACH_FINISH=
+
+# description: Reminder 1st outreach offset (HHMM). default 4800
```

Comment:
I'm not quite sure on what this one does.

Reply:
There will a companion customer implementation guide that will further explain the 4 reminder variables. The 'start' is time-of-day when reminders can start being sent to patients, 'end' is time-of-day when reminders stop sending, the 2 'offsets' are offset from appointment time when the reminder should get sent out. e.g., 48 hrs before appointment time.

Reply:
Sounds good to me. A link to the customer implementation guide when it's written up from the Code Exchange catalog page for this app and its `README.md` would probably be a worthy addition.

Reply:
Right, README.md will be the technical guide for customer's technical person. I will working on it and will refer to the (non-technical) customer implementation guide when avaiable.